code/API_definitions/qod-api.yaml

```diff
@@ -456,6 +465,15 @@ components:
           example:
             code: NOT_FOUND
             message: "Session Id does not exist"
+    Generic501:
```

Comment:
Yes.. right...   **I think we can remove this schema...!!!** As 501 (NOT_IMPLEMENTED) error code is required only for /sessions **not for** /get sessions/{sessionId} , /delete session and /notifications

because only  in create sessions  we required UE identifiers in all other request we required **session id which is not UE**   so we don't need Generic501 schema above 501 response is sufficient....

Reply:
Committed code with the changes. Kindly have a look